src/constants/index.ts

```diff
@@ -44,6 +44,16 @@ export const POOL_HIDE: { [key: string]: string[] } = {
   [SupportedNetwork.CELO]: [],
   [SupportedNetwork.BSC]: [],
 }
+
+export const START_BLOCKS: { [key: string]: number } = {
+  [SupportedNetwork.ETHEREUM]: 1,
+  [SupportedNetwork.POLYGON]: 1,
+  [SupportedNetwork.ARBITRUM]: 1,
+  [SupportedNetwork.OPTIMISM]: 1,
+  [SupportedNetwork.CELO]: 1,
```

Comment:
would these ideally be the block where the factory contract was deployed?

Reply:
0x8799e697051FC05D0BEE6BD2841A3C017f566097